api/_utils/db/db-manager.ts

```diff
@@ -244,20 +243,21 @@ export async function adoptTree(
 
 interface WaterTreeProps {
   tree_id: string;
+  time: string;
   uuid: string;
   amount: number;
   username: string;
 }
 export async function waterTree(opts: WaterTreeProps): Promise<string> {
-  const { tree_id, uuid, amount, username } = opts;
+  const { tree_id, time, uuid, amount, username } = opts;
   await pool.query(
     `
     INSERT INTO trees_watered (tree_id, time, uuid, amount, timestamp, username)
```

Comment:
General question. Why do we have a time and timestamp column. 🤦🏽

Reply:
Is time the right column? What does the frontend use?

Reply:
Good question. There were already both as I started and I wondrerd as well. I looked at the prisma schema and timestamp seemed to be a timestamp and therefore I assumed it would be the creation date. As time was set separately as a string, I assumed this was used as the watering time. However I just checked in the frontend, and there we use timestamp. So I guess we should use that? I'm confused. What do you think?

Reply:
If we use the `timestamp` column to display information we should stick to that. Way better then using strings.

Reply:
Did you also look what the charts are using?